wagtail/wagtailforms/models.py

```diff
@@ -295,7 +295,15 @@ def process_form_submission(self, form):
 
     def send_mail(self, form):
         addresses = [x.strip() for x in self.to_address.split(',')]
-        content = '\n'.join([x[1].label + ': ' + text_type(form.data.get(x[0])) for x in form.fields.items()])
+        content = []
+        for field in form:
+            value = field.value()
+            if isinstance(value, list):
+                value = ', '.join(value)
+                if len(value) == 0:
```

Comment:
Perhaps an empty value should just be exactly what your original code produced, the field label and an empty string. Then we would have to change the test accordingly so that it would not expect a string value of None.